vscode/src/commands/deploy/deploy.ts

```diff
@@ -70,7 +70,8 @@ export const processDeploy = async (
       const airdropAmount = PgConnection.getAirdropAmount();
       if (airdropAmount !== null) {
         throw new Error(
-          errMsg + `\nYou can use airdrop command to airdrop some SOL.`
+          errMsg +
+            `\nYou can use airdrop command to airdrop some SOL. \n Or you can get devnet sol from the web faucet https://faucet.solana.com/?walletAddress=${walletKp.publicKey.toString()}&amount=5 \nOr you can follow this guide to find other ways on how to get dev net sol: https://solana.com/de/developers/guides/getstarted/solana-token-airdrop-and-faucets`
```

Comment:
This line is extremely long (357 characters). Could you split it up to multiple lines like [here](https://github.com/solana-playground/solana-playground/blob/c9829073999396b58d4d06e0387bad168f1ad5a6/client/src/tutorials/create.ts#L8-L11)? Maximum of 80 characters per line is `prettier`'s default.